clients/src/main/resources/common/message/LeaveGroupRequest.json

```diff
@@ -18,11 +18,19 @@
   "type": "request",
   "name": "LeaveGroupRequest",
   // Version 1 and 2 are the same as version 0.
-  "validVersions": "0-2",
+  // Version 3 defines batch processing scheme with group.instance.id + member.id for identity
+  "validVersions": "0-3",
   "fields": [
     { "name": "GroupId", "type": "string", "versions": "0+", "entityType": "groupId",
       "about": "The ID of the group to leave." },
-    { "name": "MemberId", "type": "string", "versions": "0+",
-      "about": "The member ID to remove from the group." }
+    { "name": "MemberId", "type": "string", "versions": "0-2",
+      "about": "The member ID to remove from the group." },
+    { "name": "MemberIdentityList", "type": "[]MemberIdentity", "versions": "3+",
```

Comment:
Should we also add "nullableVersions": "3+", "default": "null" as well?

EDIT: nvm, for list fields it could just be an empty list but never should be null.

Reply:
nit: `MemberIdentityListV0` in case we want to change its version in the future as well.

Reply:
Not sure I follow here, why couldn't we just add/remove fields in the current `MemberIdentityList` elements?

Reply:
We could, it's just a naming nit :) I was reading other fields like `UpdateMetadataRequest.PartitionStatesV0` and would like to be more consistent.

Reply:
Sg, updated